stellarphot/settings/models.py

```diff
@@ -0,0 +1,48 @@
+# Objects that contains the user settings for the program.
+
+from enum import Enum
+
+from astropy import units as un
```

Comment:
Not a complaint, just curious why you switched to `units as un` instead of `units as u`, which seems more conventional?

Reply:
😬 the typing expansion program changes `u` to `you` so I used `un`. I can change that....

Reply:
Not to be difficult, but can we be consistent and use `u` for `astropy.units`?